README.md

```diff
@@ -432,3 +432,22 @@ Cron check-ins are configured in the following locations:
 2. `config/settings.yml`: Stubs out a check-in key for each cron job. Since we may not want to have a check-in for all environments, this stub key will be used and produce a null check-in.
 3. `config/settings/production.yml` in shared_configs: This contains the actual check-in keys.
 4. HB notification page: Check-ins are configured per project in HB. To configure a check-in, the cron schedule will be needed, which can be found with `bundle exec whenever`. After a check-in is created, the check-in key will be available. (If the URL is `https://api.honeybadger.io/v1/check_in/rkIdpB` then the check-in key will be `rkIdpB`).
+
+## Query Performance Instrumentation
+To log slow queries to `logs/slow_query.log`, enable the slow query instrumentation:
+```
+slow_queries:
+  enable: true
+  threshold: 500 # milliseconds
+```
+
+To instrument a query, include `InstrumentationSupport` and then annotate the query. For example:
+```
+module ReplicationService
+  include InstrumentationSupport
+
+  def num_replication_errors
+    ZipPart.annotate(sa).where.not(status: 'ok').count
```

Comment:
And, just curious: where does the `#annotate` method come from? Is that a Rails thing?

Reply:
https://edgeapi.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-annotate